test/testrunner.py

```diff
@@ -293,12 +269,15 @@ def main():
                 )
             framework = frameworks_in_images[0]
             is_neuron = "neuron" in dlc_images
-            eks_cluster_name = setup_eks_cluster(framework, is_neuron)
-
-            if not is_neuron:
-                # setup kubeflow
-                eks_utils.setup_kubeflow(eks_cluster_name)
-
+            build_context = get_build_context()
+            eks_cluster_name = f"{framework}-{build_context}"
+            
+            if eks_utils.is_eks_cluster_active(eks_cluster_name):
+                eks_utils.eks_write_kubeconfig(eks_cluster_name)
+            else:
+                print(f"EKS cluster {eks_cluster_name} is not in active state")
+                sys.exit(1)
```

Comment:
It would be better if this were to be an `Exception`.

Reply:
done